src/styles/main.scss

```diff
@@ -1 +1,10 @@
-// put code here
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+}
+
+@import "./utils/variables";
```

Comment:
You are missing the dot before the path, causing the file to not be found. It should be './utils/variables'.